geniza/corpus/templates/corpus/snippets/document_transcription.html

```diff
@@ -16,36 +16,35 @@
     {% endif %}
 {% endif %}
 {# only enable ITT panel if transcription or images are present, or we are in edit mode #}
-{% if document.has_transcription or document.iiif_urls or edit_mode %}
+{% if document.has_transcription or document.has_translation or document.iiif_urls or edit_mode %}
 
-    <section id="itt-panel" {% if not edit_mode %}data-controller="transcription" data-action="click@document->transcription#clickCloseDropdown"{% endif %}>
+    <section id="itt-panel" data-controller="ittpanel{% if not edit_mode %} transcription" data-action="click@document->transcription#clickCloseDropdown{% endif %}">
         {# NOTE: inputs must be kept as SIBLINGS of panel-container for CSS selection/controls #}
         {# images displayed by default; disable if no images are available #}
         {# translators: label for checkbox toggle to show the images panel for a document #}
-        <input type="checkbox" class="toggle" id="images-on" aria-label="{% translate 'show images' %}" {% if document.has_image or edit_mode %}checked="true" {% else %} disabled="true"{% endif %}/>
+        <input type="checkbox" class="toggle" id="images-on" data-ittpanel-target="toggle" data-action="ittpanel#clickToggle" aria-label="{% translate 'show images' %}" {% if document.has_image or edit_mode %}checked="true" {% else %}disabled="true" {% endif %}/>
```

Comment:
why is a controller needed for this?

Reply:
I think JS is needed for [the logic](https://github.com/Princeton-CDH/geniza/blob/5caaccd19550da777106022e36d36f99a999bd6a/sitemedia/js/controllers/ittpanel_controller.js#L12-L26) to determine which two should be active if a third one is clicked. Unless there's a way to uncheck checked boxes automatically with HTML/CSS!

Reply:
How did you decide on the logic of what should be hidden?

Reply:
Consultation with Gissoo, who worked with researchers to determine the logic.